unit/goto-instrument/cover_instrument.cpp

```diff
@@ -0,0 +1,20 @@
+#include <goto-instrument/cover_instrument.h>
+#include <testing-utils/use_catch.h>
+
+TEST_CASE("cover_intrument_end_of_function")
+{
+  // Arrange
+  goto_programt goto_program{};
+  goto_program.add(
+    goto_programt::make_function_call(code_function_callt({}, {})));
+  goto_program.add(
+    goto_programt::make_function_call(code_function_callt({}, {})));
+  goto_program.add(goto_programt::make_return());
```

Comment:
Suggest to make the goto program end with an `END_FUNCTION` instruction to make it more well-formed.